docs/dev/model_template_example.rb

```diff
@@ -18,8 +18,8 @@ class Dog < ActiveRecord::Base
   attr_accessor :number_of_barks
   attribute_method_prefix 'reset_'
 
-  alias_method :go_home, :save
-  alias_method :go_home!, :save!
+  alias go_home save
```

Comment:
Don't use `alias`, let it with `alias_method`. 
http://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html
http://stackoverflow.com/questions/4763121/should-i-use-alias-or-alias-method